modules/api/src/main/java/com/intuit/wasabi/api/AssignmentsResource.java

```diff
@@ -645,11 +622,94 @@ public Response clearMetadataCache(@HeaderParam(AUTHORIZATION) @ApiParam(value =
     @Produces(APPLICATION_JSON)
     @ApiOperation(value = "Get assignments metadata cache details...")
     @Timed
-    public Response getMetadataCacheDetails(@HeaderParam(AUTHORIZATION) @ApiParam(value = EXAMPLE_AUTHORIZATION_HEADER, required = true) final String authorizationHeader) {
-        UserInfo.Username userName = authorization.getUser(authorizationHeader);
-        authorization.checkSuperAdmin(userName);
-
+    public Response getMetadataCacheDetails() {
```

Comment:
Yes, this is only retrieving metadata cache info and NOT changing anything so to keep API easy to use, removed authorization like queueLength etc..